CMakeLists.txt

```diff
@@ -119,14 +119,20 @@ add_custom_target(
   COMMAND echo ${TOIT_GIT_VERSION} > "${CMAKE_BINARY_DIR}/sdk/VERSION"
 )
 
+add_subdirectory(src)
+
+enable_testing()
+add_subdirectory(tests)
+
 set(CMAKE_POLICY_DEFAULT_CMP0076 OLD)
+# There is no prefix, so this variable might be interfere with other libraries.
```

Comment:
this variable -> ? (be more specific)

be interfere -> interfere

Reply:
done. (reworded a bit).